templates/base.html

```diff
@@ -13,11 +38,68 @@
 {% block extra_head %}{% endblock %}
 </head>
 <body class="{% block body_class %}{% endblock %}">
+<script>
+    ((((window.trb || (window.trb = {})).data || (trb.data = {})).metrics || (trb.data.metrics = {})).thirdparty = {
+        pageName: 'bs:dataproject:news:maryland:salary-2107:dataproject.',
+        channel: 'news:maryland',
+        server: 'data.baltimoresun.com',
+        hier1: 'baltimoresun:news:maryland',
+        hier2: 'news:maryland',
+        prop1: 'D=pageName',
+        prop2: 'news',
+        prop38: 'dataproject',
+        prop57: 'D=c38',
+        eVar20: 'baltimoresun',
+        eVar21: 'D=c38',
+        eVar34: 'D=ch',
+        eVar35: 'D=pageName',
+        events:''
+    });  
+</script>
+
+<!-- Start: GPT Async -->
+<script type='text/javascript'>
+  var gptadslots=[];
+  var googletag = googletag || {};
+  googletag.cmd = googletag.cmd || [];
+  (function(){ var gads = document.createElement('script');
+  gads.async = true; gads.type = 'text/javascript';
+  var useSSL = 'https:' == document.location.protocol;
+  gads.src = (useSSL ? 'https:' : 'http:') + '//www.googletagservices.com/tag/js/gpt.js';
+  var node = document.getElementsByTagName('script')[0];
+  node.parentNode.insertBefore(gads, node);
+  })();
+  </script>
+
+  <script type="text/javascript">
+  googletag.cmd.push(function() {
+
+  //Adslot 1 declaration
+  gptadslots[1]= googletag.defineSlot('/4011/trb.baltimoresun/news', [[728,90]],'div-gpt-ad-785317941620082696-1').setTargeting('pos',['1']).addService(googletag.pubads());
 
-{% block content %}
-{% endblock %}
+  googletag.pubads().setTargeting('ptype',['s']);
+  googletag.pubads().enableAsyncRendering();
+  googletag.enableServices();
+  });
+</script>
 
-<div class="ft">
+<script src="http://www.baltimoresun.com/thirdpartyservice?disablenav=false&disablessor=true" async></script>
```

Comment:
It should technically be `&amp;`, although it renders either way.

Reply:
done